05/app.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+
+const root = createRoot(document.querySelector("#root"));
+
+class WeatherCheck extends React.Component {
+  constructor(props) {
+    super(props);
+    const {lat} = this.props;
+    this.state = {
+      data: `https://api.weatherbit.io/v2.0/current?key=44cf234f09214818a8e48efc2c273579&lat=${(lat="123")}&lon=-78.6382&lang=pl`,
```

Comment:
Tutaj wstawiamy te dane ``https://api.weatherbit.io/v2.0/current?key=44cf234f09214818a8e48efc2c273579&lat=${lat}&lon=${lon}&lang=pl``

PS. Nie musimy tego przechowywać w `state` bo to element, który się nie zmienia - raz ustawiamy i potem już korzystamy więc może być po prostu `this.apiUrl = ....`